src/main/java/tc/oc/pgm/listeners/ChatDispatcher.java

```diff
@@ -102,6 +103,22 @@ public void sendAdmin(Match match, MatchPlayer sender, @Nullable @Text String me
       desc = "Send a direct message to a player",
       usage = "[player] [message]")
   public void sendDirect(Match match, MatchPlayer sender, Player receiver, @Text String message) {
+    MatchPlayer matchReceiver = manager.getPlayer(receiver);
+    if (matchReceiver != null) {
+      SettingValue option = matchReceiver.getSettings().getValue(SettingKey.PM);
+
+      switch (option) {
+        case PM_OFF:
+          if (!sender.getBukkit().hasPermission(Permissions.STAFF)) {
```

Comment:
A switch-case for 1 option, feels pretty redundant
```java
if (option == PM_OFF && !sender.getBukkit().hasPermission(Permissions.STAFF)) {
  // send deny
  return;
}
```

Reply:
Yeah, I'll change that.